.github/workflows/nightly.yml

```diff
@@ -82,7 +88,7 @@ jobs:
                 upload_url: https://uploads.github.com/repos/Pure-D/serve-d/releases/20717582/assets{?name,label}
                 release_id: 20717582
                 asset_path: ./serve-d.zip
-                asset_name: serve-d_windows-nightly-$$.zip
+                asset_name: serve-d_windows-nightly-${{ matix.arch }}-$$.zip
```

Comment:
originally I made the windows release just 32 bits, what are the advantages for a 64 bit build?

I was thinking the bigger memory address space could be useful for the fibers and supporting big projects, otherwise I don't know what it will change. Will merge but would still like to know other stuff that would change with this as well.

Reply:
Directly, it fixes the issue with spawning DCD. See the issue
https://github.com/Pure-D/atom-ide-d/issues/42#issuecomment-786394763

But in general, about the architecture itself, x64 allows the program to use more memory and use faster instructions. Usually, x64 programs run faster! 32 bit systems are almost deprecated on Windows, and they run in compatibility mode.

Reply:
Here is a more in-depth explanation:
https://en.wikipedia.org/wiki/64-bit_computing#32-bit_vs_64-bit